subprojects/core-api/src/main/java/org/gradle/api/artifacts/dsl/DependencyHandler.java

```diff
@@ -444,13 +446,22 @@
     void registerTransform(Action<? super VariantTransform> registrationAction);
 
     /**
-     * Registers an artifact transform.
+     * Registers an artifact transform with a parameter.
```

Comment:
`with parameters`?

Reply:
It is only one parameter object. We can call the parameter object `parameters`.

Shall I maybe call this thing `with a parameter object`?

Reply:
Changed to `with a parameter object`.